lib/rendezvous.rb

```diff
@@ -89,7 +89,13 @@ def socket
       host, port, secret = uri.host, uri.port, uri.path[1..-1]
 
       ssl_context = OpenSSL::SSL::SSLContext.new
-      ssl_context.ssl_version = :TLSv1
+
+      if ssl_context.respond_to?(:min_version=)
+        ssl_context.min_version = :TLS1
+        ssl_context.max_version = :TLS1_2
+      else
+        ssl_context.ssl_version = :TLSv1
```

Comment:
So anything using ruby < 2.5 will still be stuck on this deprecated version, right? I think that's ok (we're targeting the fix at our own apps which should be on ruby >= 2.5) but it might be nice to acknowledge (that we're skipping this for now) in the commit message..?

Reply:
Yeah, that's my thinking. I've updated the commit message.